inductiva/tests/simulators/test_simulator.py

```diff
@@ -105,3 +107,24 @@ def test_valid_resources__mpi_simulators(simulator):
     correctly."""
 
     assert resources.MPICluster in simulator.get_supported_resources()
+
+
+@mark.parametrize("input_dir,expects_assertion_error",
+                  [(pathlib.Path().cwd(), True),
+                   (pathlib.Path().cwd() / "..", True),
+                   (pathlib.Path().cwd() / ".." / "other_dir", False)])
```

Comment:
what if `input_dir` is a grand+ parent of `cwd`. It should fail in that case as well.